telegram/ext/jobqueue.py

```diff
@@ -123,10 +155,11 @@ def tick(self):
                 continue
 
             if job.enabled:
-                self.logger.debug('Running job %s', job.name)
-
                 try:
-                    job.run(self.bot)
+                    for day in job.days:
```

Comment:
If someone would, for whatever reason, put a day twice (like `Job(..., days=(Days.mon, Days.mon))`) this would run the job twice on those days. I'm not sure this is a bug or a feature, but it could be easily changed by changing the type of `Job.days` from `tuple` to `set`. Your choice.